src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -52,7 +52,7 @@ public function __toString()
         foreach ($this->headers as $name => $values) {
             $name = implode('-', array_map('ucfirst', explode('-', $name)));
             foreach ($values as $value) {
-                $content .= sprintf("%-{$max}s %s\r\n", $name.':', $value);
+                $content .= sprintf("%-${max}s %s\r\n", $name.':', $value);
```

Comment:
It were 4 more cases with sprintf and inline variables, as ${...} structure. Should I make adjustments to {$...} then?